<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Hamzas-MacBook-Pro.local" ignored="1" name="HubSpot Functional Regression Test -- firefox" tests="6" failures="3" timestamp="2020-09-06T01:24:08 CDT" time="56.053" errors="0">
  <testcase name="@BeforeMethod setUp" time="0.056" classname="com.qa.hubspot.test.LoginPageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.qa.hubspot.base.BasePage.init_driver(BasePage.java:46)
at com.qa.hubspot.test.LoginPageTest.setUp(LoginPageTest.java:56)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@AfterMethod tearDown" time="0.001" classname="com.qa.hubspot.test.LoginPageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.qa.hubspot.test.LoginPageTest.tearDown(LoginPageTest.java:119)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase name="@BeforeMethod setUp" time="0.008" classname="com.qa.hubspot.test.LoginPageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.qa.hubspot.base.BasePage.init_driver(BasePage.java:46)
at com.qa.hubspot.test.LoginPageTest.setUp(LoginPageTest.java:56)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@AfterMethod tearDown" time="0.001" classname="com.qa.hubspot.test.LoginPageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.qa.hubspot.test.LoginPageTest.tearDown(LoginPageTest.java:119)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase name="@BeforeMethod setUp" time="0.018" classname="com.qa.hubspot.test.LoginPageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.qa.hubspot.base.BasePage.init_driver(BasePage.java:46)
at com.qa.hubspot.test.LoginPageTest.setUp(LoginPageTest.java:56)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@AfterMethod tearDown" time="0.001" classname="com.qa.hubspot.test.LoginPageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.qa.hubspot.test.LoginPageTest.tearDown(LoginPageTest.java:119)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase name="verifyPageTitleTest" time="0.0" classname="com.qa.hubspot.test.LoginPageTest">
    <skipped/>
  </testcase> <!-- verifyPageTitleTest -->
  <testcase name="verifySignUpLinkTest" time="0.0" classname="com.qa.hubspot.test.LoginPageTest">
    <skipped/>
  </testcase> <!-- verifySignUpLinkTest -->
  <testcase name="loginTest" time="0.0" classname="com.qa.hubspot.test.LoginPageTest">
    <skipped/>
  </testcase> <!-- loginTest -->
  <testcase name="verifyHomePageTitleTest" time="20.398" classname="com.qa.hubspot.test.HomePageTest">
    <failure type="org.openqa.selenium.TimeoutException" message="Expected condition failed: waiting for title to be &amp;quot;Account Setup | HubSpot&amp;quot;. Current title: &amp;quot;HubSpot Login&amp;quot; (tried for 20 second(s) with 500 milliseconds interval)
Build info: version: &amp;apos;3.141.59&amp;apos;, revision: &amp;apos;e82be7d358&amp;apos;, time: &amp;apos;2018-11-14T08:17:03&amp;apos;
System info: host: &amp;apos;Hamzas-MacBook-Pro.local&amp;apos;, ip: &amp;apos;fe80:0:0:0:1c6d:233e:f8cd:15c5%en0&amp;apos;, os.name: &amp;apos;Mac OS X&amp;apos;, os.arch: &amp;apos;x86_64&amp;apos;, os.version: &amp;apos;10.15.6&amp;apos;, java.version: &amp;apos;1.8.0_261&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 85.0.4183.83, chrome: {chromedriverVersion: 85.0.4183.87 (cd6713ebf92fa..., userDataDir: /var/folders/mr/2v0wmr810qg...}, goog:chromeOptions: {debuggerAddress: localhost:51109}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:virtualAuthenticators: true}
Session ID: 3d5ca550acd59b6ea3389097b80f2821">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for title to be "Account Setup | HubSpot". Current title: "HubSpot Login" (tried for 20 second(s) with 500 milliseconds interval)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'Hamzas-MacBook-Pro.local', ip: 'fe80:0:0:0:1c6d:233e:f8cd:15c5%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.15.6', java.version: '1.8.0_261'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 85.0.4183.83, chrome: {chromedriverVersion: 85.0.4183.87 (cd6713ebf92fa..., userDataDir: /var/folders/mr/2v0wmr810qg...}, goog:chromeOptions: {debuggerAddress: localhost:51109}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:virtualAuthenticators: true}
Session ID: 3d5ca550acd59b6ea3389097b80f2821
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:95)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:272)
at com.qa.hubspot.util.ElementUtil.waitForTitlePresent(ElementUtil.java:40)
at com.qa.hubspot.test.HomePageTest.verifyHomePageTitleTest(HomePageTest.java:44)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- verifyHomePageTitleTest -->
  <testcase name="verifyHomePageHeaderTest" time="10.019" classname="com.qa.hubspot.test.HomePageTest">
    <failure type="java.lang.AssertionError" message="expected [Setup Guide] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [Setup Guide] but found [null]
at com.qa.hubspot.test.HomePageTest.verifyHomePageHeaderTest(HomePageTest.java:54)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- verifyHomePageHeaderTest -->
  <testcase name="getAccountNameTest" time="10.015" classname="com.qa.hubspot.test.HomePageTest">
    <failure type="java.lang.AssertionError" message="expected [Kardesler] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [Kardesler] but found [null]
at com.qa.hubspot.test.HomePageTest.getAccountNameTest(HomePageTest.java:61)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- getAccountNameTest -->
  <testcase name="login_invalidTestCase" time="0.0" classname="com.qa.hubspot.test.LoginPageTest">
    <ignored/>
  </testcase> <!-- login_invalidTestCase -->
</testsuite> <!-- HubSpot Functional Regression Test - firefox -->
